rules/feature/targettype.yaml

```diff
@@ -96,3 +96,11 @@
 		RequiresForceFire: true
 	Targetable@TargetTypeNoAutoTarget:
 		TargetTypes: NoAutoTarget
+
+^FeatureTargettypeFloraAir:
```

Comment:
Then you probably want the `^FeatureTargettypeNoAutoTarget` one which disables auto target (force fire is still possible though). If you don't want them to be targeted at all, just remove those Inherits completely.